modules/core/src/main/scala/GuardrailAnalysis.scala

```diff
@@ -0,0 +1,16 @@
+package com.twilio.guardrail
+package sbt
+
+import _root_.sbt._
+import _root_.sbt.util.CacheImplicits._
+import sjsonnew.{ :*:, LList, LNil}
```

Comment:
TIL this exists inside sbt's dependencies, neat!

Reply:
Thanks to @eed3si9n !